variables.tf

```diff
@@ -78,6 +78,18 @@ variable "apiary_managed_schemas" {
   default     = []
 }
 
+variable "apiary_delete_schemas" {
+  description = "List of maps, each map contains schema name which will be deleted from the Hive metastore, and its corresponding S3 bucket deleted."
```

Comment:
This says S3 bucket will be deleted but docs above say S3 data won't be deleted so I'm a bit confused as to what it is. Also, why is this a list of maps and not of strings containing db names? What are the map keys and values?

Reply:
Good catch, thanks.  Originally I was going to delete the s3 buckets also, but after talking to Raj, decided it was better to never do automated deletion of actual data.  S3 data can be cleaned up manually if necessary.

The entry is a list of maps (instead of a list of strings) for two reasons:

1.  So that entries from `apiary_managed_schemas` can be easily copy/pasted to this entry when schemas to delete are indentified.
2. More importantly, it is a map to reserve the ability to add extra properties to each entry to control future delete-time functionality.

Reply:
Ack on the maps, I see you have documented that now too which makes it clearer as to why this type.